<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>9</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayout2"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayout1">
     <property name="leftMargin">
      <number>5</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClear"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionFrom_File"/>
   </widget>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>Command</string>
    </property>
    <addaction name="actionEncrypt"/>
    <addaction name="actionDecrypt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuCommand"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFrom_File">
   <property name="text">
    <string>From File</string>
   </property>
  </action>
  <action name="actionEncrypt">
   <property name="text">
    <string>Encrypt</string>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="text">
    <string>Decrypt</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
